tests/user/stake.test.ts

```diff
@@ -38,15 +42,13 @@ test("stake(1000)", () => {
         "timestamp",
         txTimestamp.toString()
     )
-
-    clearStore()
 })
 
 test("stake(1000) while maturing", () => {
     let address = "0x0000000000000000000000000000000000000000"
 
     // create a user with 500 maturing
-    let user = new User(address)
+    let user = loadOrCreate(Address.fromString(address))
```

Comment:
Since https://github.com/graphprotocol/graph-cli/commit/629176c3c806adb02876f76c2ba1c3dac6933450 the code generation of graph-cli does not assign default values for entities, so we need to initialize ourselves all properties.